packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1183,23 +1809,23 @@ class CupertinoDialogAction extends StatelessWidget {
     // wrap instead of ellipsizing. We are consciously not implementing that
     // now due to complexity.
     final Widget sizedContent = _isInAccessibilityMode(context)
-      ? _buildContentWithAccessibilitySizingPolicy(
-          textStyle: style,
-          content: child,
-        )
-      : _buildContentWithRegularSizingPolicy(
-          context: context,
-          textStyle: style,
-          content: child,
-        );
+        ? _buildContentWithAccessibilitySizingPolicy(
```

Comment:
Extra indentation like this is fine either way I think.

Reply:
Yes, I saw in cases like this both 2 and 4 space indents are used. Since I prefer 4 I used it.